src/social_db.rs

```diff
@@ -5,5 +5,6 @@ pub const SOCIAL_DB: &str = "social.near";
 
 #[ext_contract(ext_social_db)]
 pub trait SocialDB {
-    fn set(&mut self, mut data: Value);
+    // interface from: https://github.com/NearSocial/social-db/blob/39016e654739b0a3e8cb7ffaea4b03157c4aea6e/contract/src/api.rs#L135
+    fn set(&mut self, #[allow(unused_mut)] mut data: Value);
```

Comment:
Can we just drop the `mut` before `data`?

Reply:
I think that's not right. The cross contract caller should faithfully follows (ideally import) the interface of callee contract. I looked at https://github.com/NearSocial/social-db/blob/39016e654739b0a3e8cb7ffaea4b03157c4aea6e/contract/src/api.rs#L135, the `mut` cannot be dropped there.

Reply:
`mut` is not part of the interface, unlike `&` or `&mut`, so I would assume it is fine to drop it.

Reply:
That's good to know! I'll drop it then.